Source/DafnyServer/CounterExampleGeneration/DafnyModel.cs

```diff
@@ -154,6 +160,64 @@ public class DafnyModel {
       return result;
     }
 
+    /// <summary>
+    /// Registered all values of basic types specified by the model in
+    /// the reservedValues map;
+    /// </summary>
+    private void RegisterReservedValues() {
```

Comment:
My personal preference would be extracting the steps of this method into distinct methods, for example:

- RegisterCharType
- RegisterIntType
- RegisterBoolType
- etc.

But current style is already common within the compiler.

Reply:
Done!